ui/src/styles/var/_fonts.scss

```
@@ -0,0 +1,17 @@
+$xxx-small: px-to-rem(10);
+$xx-small: px-to-rem(12);
+$x-small: px-to-rem(14);
+$small: px-to-rem(16);
+$medium: px-to-rem(20);
+$large: px-to-rem(24);
+
+// Font weights
+$regular: 400;
+$bold: 600;
+
+$line-height: 1.5;
```

Stylelint (17.7.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)